recipes/kdbindings/all/conanfile.py

```diff
@@ -13,7 +13,7 @@ class KDBindingsConan(ConanFile):
     homepage = "https://github.com/KDAB/KDBindings"
     url = "https://github.com/conan-io/conan-center-index"
     package_type = "header-library"
-    settings = "os", "arch", "compiler", "build_type"
+    settings = "compiler"
```

Comment:
```suggestion
    settings = "os", "arch", "compiler", "build_type"
```
The convention in Conan 2.0 is to always set all settings and clear them in `package_id()` instead.

Reply:
@bog-dan-ro Please, avoid changing settings, so we can use any settings in the recipe. The package ID will not change due `self.info.clear()`.